pixels/canvas.py

```diff
@@ -45,12 +45,8 @@ async def _populate_cache(self, conn: Connection) -> None:
 
         cache = bytearray(Sizes.WIDTH * Sizes.HEIGHT * 3)
 
-        sql = (
-            "SELECT x, y, rgb "
-            "FROM current_pixel "
-            "WHERE x < $1 "
-            "AND y < $2"
-        )
+        sql = "SELECT x, y, rgb FROM current_pixel WHERE x < $1 AND y < $2"
```

Comment:
Might as well be consistent with the usage of semicolons.
```suggestion
        sql = "SELECT x, y, rgb FROM current_pixel WHERE x < $1 AND y < $2;"
```

Reply:
A bunch of places used these, and a bunch of places didn't.

I made all of these consistent in https://github.com/python-discord/pixels/pull/106/commits/a4b47d7b88b57da0eaa467553cafe52bfb2448a5